### ISSUE TITLE
Update documentation in light of recent chocolatey package.

### ISSUE THREAD

**Issue by springcomp** [springcomp/optimized-azerty-win#103]
_

**Comment by vincentremond** [comment]
I'll do that as soon as my package is validated by a moderator.

**Comment by springcomp** [comment]
@vincentremond did you notice that the package seems to have been rejected by chocolatey?

**Comment by vincentremond** [comment]
Yeah I have been notified of the rejection, they requested me to change the id of the package. But when notified me of that, I had not any free time left to do it :(

Here is the message they've sent me :
> Hi,
> the id of this package seems a bit "unhandy" / "bulky" to me .. would you mind renaming/repushing this as optimized-azerty-win - so it matches the actual software name?
> please let us know, so we can move forward / reject this package.
> 
> thanks

As I am confined home, I might find some time to publish the package again…

**Comment by springcomp** [comment]
> As I am confined home, I might find some time to publish the package again…

We all are, unfortunately...
Good luck and stay safe!

**Comment by springcomp** [comment]
Hello @vincentremond I’m glad to announce the new [Release 1.8.0.0](https://github.com/springcomp/optimized-azerty-win/releases/tag/v1.8.0.0) following fun discussion with Microsoft and members from the standards committee.

Do you still maintain the chocolatey package ?
Would there still be possible to publish a new version ?

**Comment by vincentremond** [comment]
Hello @springcomp , Sorry but I have not found the time to maintain this package, would you want to take over this repo ?
